sharedbuffers/mapped_struct.py

```diff
@@ -2614,9 +2607,9 @@ def __repr__(self):
     _cpython = sys.subversion[0] == 'CPython'
 is_cpython = cython.declare(cython.bint, _cpython)
 
-XXPRIME_1 = cython.declare(cython.long, 11400714785074694791)
-XXPRIME_2 = cython.declare(cython.long, 14029467366897019727)
-XXPRIME_5 = cython.declare(cython.long, 2870177450012600261)
+XXPRIME_1 = cython.declare(cython.size_t, 11400714785074694791)
```

Comment:
Py_hash_t if it's for python hashes, IIRC.

Reply:
But it's unsigned, `uhash` not `hash`, and `Py_size_t` doesn't work either.